files/check_dotnet_createdump.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+POD_NAME="$1"
+shift
+
+EXPECTED_RESULT="Dump successfully written"
+
+test -z "${POD_NAME}" && echo "'pod_name' variable is not defined in yml" && exit 1
+
+# Get the runtime version
+# Technically, this gets the ASP.NET version, but they should be the same.
+# If it is ever the case that they are not, you could grep for Microsoft.NETCore.App first
+VERSION=$(oc exec ${POD_NAME} -t -- /bin/bash -c "dotnet --list-runtimes | grep -P -o -m 1 [0-9].[0-9].[0-9]\+")
+echo ${VERSION}
+# Get the executable path
+DOTNET_HOME=$(oc exec ${POD_NAME} -t -- /bin/bash -c 'dirname $(readlink -f $(command -v dotnet))')
+echo ${DOTNET_HOME}
+
+# It seems the dotnet application is always PID 1 in the container.
+OUTPUT=$(oc exec ${POD_NAME} -t -- /bin/bash -c "${DOTNET_HOME}/shared/Microsoft.NETCore.App/${VERSION}/createdump 1 2>&1")
```

Comment:
```suggestion
# Run the 'createdump' executable from the runtime folder on the .NET application (pid 1).
OUTPUT=$(oc exec ${POD_NAME} -t -- /bin/bash -c "/usr/lib64/dotnet/shared/Microsoft.NETCore.App/*/createdump 1 2>&1")
```

Reply:
This is a bit shorter.

We can throw in `$(dirname $(readlink -f $(command -v dotnet)))` instead of `/usr/lib64/dotnet` if you prefer that.

I've also added the `--full` argument because that is in the solution article.

Reply:
If we can go with a single command then we can, theoretically, not even require a new execution file and use the 'oc exec' one I'd already made. I'm testing this now.

Reply:
Also, this same process is done in the RPM tests, and I was kind of working under the assumption that the dotnet home might be different between architectures.

If that is the case, then technically, that doesn't affect these tests as they are currently only run on x86_64, but I'd like to keep the tests setup to be able to be run against all architectures.

Reply:
Ok, I've been able to remove the new test file and use the existing one, and also still check for the dotnet homedir in the single command.